lib/file-perms.c

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2012 BalaBit IT Ltd, Budapest, Hungary
```

Comment:
I thought the patch is the result of the script. When would those changes
be applied?
On Dec 2, 2015 5:56 PM, "Tamas Nagy" <notifications@github.com> wrote:

In lib/file-perms.c
<https://github.com/balabit/syslog-ng/pull/822#discussion_r46440627>:

> @@ -1,6 +1,6 @@
>  /*
>   * Copyright (c) 2012 BalaBit IT Ltd, Budapest, Hungary

@bazsi <https://github.com/bazsi> I didn't bother to change it at this
location, because the specified script will automatically correct all
occurrences, including this one. Actually I did bother and corrected it by
hand, but I stashed all manual corrections and decided to express it in
sed. If you support such an upgrade, I'll submit the result of that script
in a different PR.

—
Reply to this email directly or view it on GitHub
<https://github.com/balabit/syslog-ng/pull/822/files#r46440627>.